frontend/mulighetsrommet-veileder-flate/src/mock/worker.ts

```diff
@@ -1,13 +1,14 @@
 import { setupWorker } from 'msw';
 import { apiHandlers } from './api/apiHandlers';
-import { veilarbpersonflateHandlers } from './api/veilarbpersonflateHandlers';
+import { veilarbdialogHandlers, veilarbpersonflateHandlers } from "./api/eksterneSystemerHandlers";
 
 // This configures a Service Worker with the given request handlers.
 
 export function initializeMockWorker() {
   const handlers = [
     ...(import.meta.env.VITE_MULIGHETSROMMET_API_MOCK === 'true' ? apiHandlers : []),
-    ...(import.meta.env.VITE_VEILARBPERSONFLATE_MOCK === 'true' ? veilarbpersonflateHandlers : []),
+    ...(import.meta.env.VITE_VEILARBPERSONFLATE_MOCK === 'true' ? veilarbpersonflateHandlers: []),
+    ...(import.meta.env.VITE_VEILARBPERSONFLATE_MOCK === 'true' ? veilarbdialogHandlers : []),
```

Comment:
Skulle vi tatt jobben og renamet `VITE_VEILARBPERSONFLATE_MOCK` til et mer generelt flagg for alle eksterne tjenester?